kitsune/sumo/static/sumo/less/questions.less

```diff
@@ -317,6 +317,7 @@ h2 {
 
   .question-tools {
     margin: 0 0 20px 130px;
+    padding: 0 20px;
```

Comment:
i've borrowed this from our current .answer-tools styling. should i change it in line 127 as well?